.travis.yml

```diff
@@ -14,7 +14,7 @@ before_script:
   - if [ ! -z "$(git status --porcelain)" ]; then git diff; echo -e "\nTest files changed. Did you run ./gradlew generateTests?"; exit 1; fi
 
 script:
-  - cd wire-library; ../gradlew build; cd ..
+  - cd wire-library && ../gradlew build && cd ..
```

Comment:
You can also run this in a subshell with `$(cd wire && ../gradlew build)`

Reply:
why not `./gradlew -p wire-library`?

Reply:
looks like I could do something like
```yaml
script:
  - ./gradlew -p wire-library build
  - ./gradlew -p . build
```